data/json/items/tools.json

```diff
@@ -2886,7 +2886,7 @@
     "flags": [ "FISH_GOOD" ]
   },
   {
-    "id": "flamable_arrow",
```

Comment:
These aren't user facing, and changing them without migration code will break saves.
Same for every json "id" field.